tests/stub/driver_parameters/test_bookmark_manager.py

```diff
@@ -474,16 +474,182 @@ def test_should_resolve_database_name_with_system_bookmarks(self):
     def _new_driver(self, bookmark_manager_config=None):
         if bookmark_manager_config is None:
             bookmark_manager_config = Neo4jBookmarkManagerConfig()
-
         uri = "neo4j://%s" % self._router.address
         auth = types.AuthorizationToken("basic", principal="neo4j",
                                         credentials="pass")
         return Driver(
             self._backend,
             uri, auth,
-            bookmark_manager_config=bookmark_manager_config
+            bookmark_manager_config=bookmark_manager_config)
```

Comment:
```suggestion
        return Driver(self._backend, uri, auth,
                      bookmark_manager_config=bookmark_manager_config)
```
or just move the closing parenthesis back onto its own line :innocent: